services/security/identity/identity.go

```diff
@@ -27,6 +27,14 @@ func getIdentity(id *wssdcloudsecurity.Identity) *security.Identity {
 		clitype = auth.LoadBalancer
 	}
 
+	tags := make(map[string]*string)
```

Comment:
Let's try to use the same patter 
https://github.com/microsoft/moc-sdk-for-go/blob/main/services/compute/virtualmachine/wssd.go#L290

Reply:
fixed